docker/run_local_docker.sh

```diff
@@ -73,11 +73,19 @@ then
       rm -rf $PWD/docker-volume
     fi
 fi
+
+if [[ $VIRTUAL_ENV == "" ]]; 
+then 
+  echo "Please Use Virtual Environment and make sure to generate Pydantic Models"; 
```

Comment:
not exiting here as chances are users might have already generated the models based on instructions from docs.